.travis.yml

```diff
@@ -23,6 +23,10 @@ after_script:
 env:
   global:
   - PGPORT=5433
-  - YEAR=2018
+  - YEAR=2019
   - SIGNING_URL="http://127.0.0.1:4000/#/{token}"
-  - OPENFISCA_ORIGN="https://govtnz-rates-rebate-of.herokuapp.com/" 
+  # TODO Update to new open fisca response
+  #- OPENFISCA_ORIGN="https://govtnz-rates-rebate-of.herokuapp.com/calculate"
+  - OPENFISCA_ORIGIN="https://openfisca.ratesrebates.services.govt.nz/calculate"
```

Comment:
Gotcha here! I had to switch back to the old openfisca. We need to either remove calculation from the backend, or update it read the new format.